olcPGEX_AnimatedSprite.h

```diff
@@ -98,8 +100,9 @@ namespace olc
 		std::string state;
 		std::map<std::string, std::vector<olc::Sprite*>> multiFrames;
 		std::map<std::string, std::vector<olc::vi2d>> singleFrames;
+		std::map<std::string, float> frameDurations;
+		const float DEFAULT_FRAME_DURATION = 0.1f;
```

Comment:
I don't really like the idea of this being a const. I'm happy for it to stay public, though, so that the default 'global' frame duration can be set. As mentioned, I was thinking about letting that be changed anyway.